src/pages/Opportunities.jsx

```diff
@@ -8,12 +8,43 @@ import SearchOpportunities from "./SearchOpportunities";
 import MapOpportunities from "./MapOpportunities";
 
 function Opportunities() {
-  const [filteredOpportunities, setFilteredOpportunities] = useState(
-    opportunities
-  );
+  const [opportunities, setOpportunities] = useState([]);
+  const [filteredOpportunities, setFilteredOpportunities] = useState([]);
+
+  useEffect(() => {
+    // Fetch data from the server
+    const fetchOpportunities = async () => {
+      try {
+        const response = await fetch("http://localhost:8000/api/v1/events/");
```

Comment:
Instead of hardcoding the base URL (http://localhost:8000), store this in an env variable so it's easy to switch between the local and deployed versions of the backend